satellite/repair/repairer/segments.go

```diff
@@ -101,7 +102,7 @@ func (repairer *SegmentRepairer) Repair(ctx context.Context, path storj.Path) (s
 
 	numHealthy := len(pieces) - len(missingPieces)
 	// irreparable piece, we need k+1 to detect corrupted pieces
-	if int32(numHealthy) < pointer.Remote.Redundancy.MinReq+1 {
+	if int32(numHealthy) < pointer.Remote.Redundancy.MinReq {
```

Comment:
is comment up to date when `+1` is removed?

Reply:
it is updated now